aptos-move/aptos-vm/src/keyless_validation.rs

```diff
@@ -81,6 +84,25 @@ pub(crate) fn get_groth16_vk_onchain(
     get_resource_on_chain::<Groth16VerificationKey>(resolver)
 }
 
+pub(crate) fn get_vk_map(
+    resolver: &impl AptosMoveResolver,
+) -> Result<HashMap<String, PreparedVerifyingKey<Bn254>>, VMStatus> {
+    let SetupsVKs { setups } = get_resource_on_chain::<SetupsVKs>(resolver)?;
```

Comment:
Can we use OnChainConfig trait here and use fetch_config? You should not need a resolver but can use anything that implements ConfigStorage, right?